cmd/snap/cmd_help.go

```diff
@@ -208,19 +208,23 @@ var helpCategories = []helpCategory{
 	}, {
 		Label:       i18n.G("Permissions"),
 		Description: i18n.G("manage permissions"),
-		Commands:    []string{"connections", "interface", "connect", "disconnect"},
+		Commands:    []string{"connect", "connections", "disconnect", "interface"},
 	}, {
 		Label:       i18n.G("Snapshots"),
 		Description: i18n.G("archives of snap data"),
-		Commands:    []string{"saved", "save", "check-snapshot", "restore", "forget"},
+		Commands:    []string{"check-snapshot", "forget", "restore", "save", "saved"},
 	}, {
-		Label:       i18n.G("Other"),
-		Description: i18n.G("miscellanea"),
-		Commands:    []string{"version", "warnings", "okay", "ack", "known", "model", "create-cohort", "recovery", "reboot"},
+		Label:       i18n.G("Status"),
+		Description: i18n.G("information output and support"),
+		Commands:    []string{"known", "okay", "version", "warnings"},
+	}, {
+		Label:       i18n.G("Devices"),
+		Description: i18n.G("system management"),
+		Commands:    []string{"model", "prepare-image", "reboot", "recovery"},
```

Comment:
I agree with Ian here, maybe we can hide "prepare-image", it's really a bit internal.